components/apiserver-proxy/README.md

```diff
@@ -0,0 +1,49 @@
+# API Server Proxy
+
+## Overview
+
+The Kyma API Server Proxy is a core component that secure access to the kubernetes apiserver using JWT authentication. 
```

Comment:
The Kyma API Server Proxy is a core component that uses JWT authentication to secure access to the Kubernetes API server.  It is based on the `kube-rbac-proxy`(https://github.com/brancz/kube-rbac-proxy) project. This [Helm chart](/resources/core/charts/apiserver-proxy/Chart.yaml) outlines the component's installation.